### PR TITLE
Feature/theme hook

### PR DESCRIPTION
**Pull request by ryansnorek**
## Description

This PR is for 2 hooks to help clean up the codebase // `useLocalStorage` and `useTheme`.
The `useTheme` hook can replace the `DarkModeToggle` component and 3 blocks of code in the `SideBar` and `PendingApproval`

[Video recording
](https://www.loom.com/share/c6a83fd5d183474db9bab2c3cb27582d)

[Trello card
](https://trello.com/c/rpusIK67)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/hooks/useLocalStorage.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export default function useLocalStorage(key, initValue) {
+  const [storedValue, setStoredValue] = useState(() => {
+    return localStorage.getItem(key) || initValue;
+  });
+  const setValue = value => {
+    setStoredValue(value);
+    localStorage.setItem(key, value);
+  };
+  return [storedValue, setValue];
```
Comment (MrJacobSullivan) [inline src/hooks/useLocalStorage.js:11]: This looks really good!

File: src/hooks/useTheme.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  }, []); //eslint-disable-line
+
+  useEffect(
+    function setAntStylesheetTheme() {
+      document.head.querySelector('#antd-stylesheet').href = stylesheets[theme];
+    },
+    [theme]
+  );
+
+  return [toggleTheme];
```
Comment (MrJacobSullivan) [inline src/hooks/useTheme.js:44]: Does this have to return an array or could we just return `toggleTheme` by itself? Doing so means we wouldn't have to catch it in an array i.e. `const [toggleTheme] = useTheme()` vs `const toggleTheme = useTheme()`.

**Review by MrJacobSullivan — APPROVED** [review]

File: src/hooks/useTheme.js (hunk 2 above)
Comment (ryansnorek) [reply src/hooks/useTheme.js]: Good catch. Updated this to return `toggleTheme` in latest commit

**Review by JohnMBarnett — APPROVED** [review]
Really good implementation that cuts out a lot of unnecessary code!

**Pull request merged**